### ISSUE TITLE
Method or pretrained weight for CIFAR10-DVS dataset

### ISSUE THREAD

**Issue by HAMBUK** [ECNU-Cross-Innovation-Lab/Mamba-Spike#4]
Hello, I am currently downloaded your works and run train of CIFAR10-DVS dataset.

Test accuracy is around 40~50% in 140 Epochs. It seems there will be quite gap between the paper results.

Can you kindly provide pre trained weights or parameter setting that have been used?

Thanks.

**Comment by D-ST-Sword** [comment]
To more precisely locate your problem, could you provide the following information
What system did you use when running the experiment
Did you use an nvidia GPU when running the experiment? If it is an nvidia GPU, what is the specific model?
How much time did it take you to run the experiment for 140 epochs? How much time does it take to complete each epoch?
What hyperparameters did you use when running the experiment?

**Comment by HAMBUK** [comment]
Hello, I am using NVIDIA RTX Pro 6000 Blacwell 96GB GPU Environment for experiment.

I used train.py with learning rate of 1e-3. (I think you updated the train codes now.)

It took about 3days for 140epochs with 32 batch size (Training resource was about 86gb of vram).

And best test accuracy was 45.40%. I will test with your updated codes and let you know.

**Comment by HAMBUK** [comment]
Can I kindly ask why the target_accuracy for CIFAR10-DVS dataset is set to 78.9%? It seems there's quite a lot of gap between 92.5% in results in paper.

Thank you for your hard work!

**Comment by D-ST-Sword** [comment]
We will conduct experiments using the hyperparameter Settings you provided to see if we encounter the problems you mentioned earlier.

78.9% is an indicator mentioned in a paper that proposed a lightweight method. We used a lightweight version to attempt a comparison. To avoid misleading in the future, we have deleted this description

**Comment by HAMBUK** [comment]
I see, is it possible for me to use original model before using lightweight technique? (Which achieved 92.5% of test accuracy in paper)
